deployment_scripts/config.ps1

```diff
@@ -2,45 +2,48 @@
 $MONKEY_FOLDER_NAME = "infection_monkey"
 # Url of public git repository that contains monkey's source code
 $MONKEY_GIT_URL = "https://github.com/guardicore/monkey"
+$MONKEY_RELEASES_URL = $MONKEY_GIT_URL + "/releases"
+$MONKEY_LATEST_VERSION = "1.7.0"
+$MONKEY_DOWNLOAD_URL = $MONKEY_RELEASES_URL + "/download/" + $MONKEY_LATEST_VERSION + "/"
```

Comment:
Bad approach, because we'll have to remember yet another thing to update. Pls use https://api.github.com/repos/guardicore/monkey/releases/latest and grep `html_url` value

Reply:
@VakarisZ 

I can do what you suggest but I have something else to think about
Why are we downloading the monkey binaries from a tag if we're allowing monkey development? What purpose does it serve?

Reply:
Island development, testing, building from source?

Reply:
And for that, it's good enough to use the latest binaries despite them maybe/maybe not matching the island?

Reply:
I think getting the binaries should be an optional step (perhaps not on by default), and that getting the latest from GH should be the option we go for. If someone changes the Monkey code they're going to need to compile Monkeys anyway.

Reply:
@VakarisZ I'm going with this.
For now it'll be an optional step and on by default to avoid breaking anything.

Reply:
(That way we can just delete it for now, open a new issue to write that script, and delay it)